docs/hsa/hsa.rst

```diff
@@ -0,0 +1,178 @@
+.. doctest-skip-all
+
+.. _astroquery.hsa:
+
+*******************************************
+Herschel Science Archive (`astroquery.hsa`)
+*******************************************
+
+Herschel was the fourth cornerstone in ESA's Horizon 2000 science programme, designed to observe the 'cool' universe. 
+It performed photometry and spectroscopy in the poorly explored 55-670 µm spectral range with a 3.5 m diameter 
+Cassegrain telescope, providing unique observing capabilities and bridging the gap between earlier infrared space 
+missions and groundbased facilities. Herschel successfully performed ~37000 science observations and ~6600 science 
+calibration observations which are publicly available to the worldwide astronomical community through the Herschel Science Archive.
+
+This package allows the access to the `Herschel Science Archive <http://archives.esac.esa.int/hsa/whsa/>`__.
+
+========
+Examples
+========
+
+------------------------------
+1. Getting Herschel data
+------------------------------
+
+.. code-block:: python
+
+  >>> from astroquery.hsa import HSA 
+  >>>
+  >>> HSA.download_data(observation_id='1342195355',retrieval_type='OBSERVATION', instrument_name='PACS')
+  Downloading URL http://archives.esac.esa.int/hsa/whsa-tap-server/data?&retrieval_type=OBSERVATION&observation_id=1342195355&instrument_name=PACS to 1342195355.tar ... [Done]
+  '1342195355.tar'
+
+This will download the product of the observation '1342195355' with the instrument 'PACS' and 
+it will store them in a tar called '1342195355.tar'. The parameters available are detailed in the API.
+
+For more details of the parameters check the section 6 of the 'Direct Product Access using TAP' in the 'HSA users guide' at:
+		'http://archives.esac.esa.int/hsa/whsa/'
+
+-------------------------------
+2. Getting Observation Product
+-------------------------------
+
+.. code-block:: python
+
+  >>> from astroquery.hsa import HSA 
+  >>>
+  >>> HSA.get_observation('1342195355', instrument_name='PACS')
+  Downloading URL http://archives.esac.esa.int/hsa/whsa-tap-server/data?&retrieval_type=OBSERVATION&observation_id=1342195355&instrument_name=PACS to 1342195355.tar ... [Done]
+  '1342195355.tar'
+
+This will download the product of the observation '1342195355' with the instrument 'PACS' and 
+it will store them in a tar called '1342195355.tar'. The parameters available are detailed in the API.
+
+For more details of the parameters check the section 6.2 of the 'Direct Product Access using TAP' in the 'HSA users guide' at:
+		'http://archives.esac.esa.int/hsa/whsa/'
+
+-------------------------------
+3. Getting Herschel Postcard
```

Comment:
This section also needs description - what is a postcard?  Is it a "cutout" or "quicklook"?  What does that mean?  Is it usable for science?  Again, this could be answered with a link to the right part of the archive, but it would be helpful to have descriptive text here